nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -71,6 +73,19 @@ in {
       description = "Which package to use for the Nextcloud instance.";
       relatedPackages = [ "nextcloud17" "nextcloud18" "nextcloud19" ];
     };
+    user = mkOption {
```

Comment:
What is the purpose of a configurable `user` option?

If the user is configurable then we should follow the pattern of conditionally provisioning users only if the default value is specified:
```
users.users.nextcloud = optionalAttrs (cfg.user == "nextcloud") {
  ...
};
```

Reply:
Actually the user doesn't really need to be configurable. Sharing the group with other services which need to access the same files should be sufficient. So I will revert this change.